Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -155,6 +155,9 @@ The [zone](../../Storage-Configuration/Object-Storage-RGW/ceph-object-multisite.
 
 ## Hosting Settings
 
+!!! warning
```

Comment:
I'm inclined to suggest we wait until the discussion in #14034 settles before we declare this experimental. We have had an upstream user confirm it worked for them. But it depends on our definition of experimental. Let's discuss.